examples/virtual/src/main.tsx

```
@@ -0,0 +1,41 @@
+
+    (async () => {
+      // @ts-ignore
+      const result = await import("virtual")
+      console.log("result", result)
+    })()
+  }, [])
+
+  return (
+    <>
+      <div>
+        <a href="https://farmfe.org/" target="_blank">
```

Biome (1.9.4)

[error] Avoid using target="_blank" without rel="noreferrer".

Opening external links in new tabs without rel="noreferrer" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noreferrer" attribute.


(lint/a11y/noBlankTarget)